src/utils/convertFile.ts

```diff
@@ -9,6 +9,7 @@ import { unoserver } from './unoserver.js'
 export async function convertFile(
 	srcPath: string,
 	format: string,
+	filter?: string,
```

Comment:
Could you please make the third argument as an options object? Like this:

```ts
srcPath: string,
format: string,
options?: {  filter?: string }
```

Reply:
Done